src/Microsoft.HttpRepl.IntegrationTests/Commands/CommandHelper.cs

```diff
@@ -0,0 +1,79 @@
+using System.Collections.Generic;
```

Comment:
nit: add copyright comments at the top of any new files (or old ones that are missing them)

Reply:
Added copyright comments to files in test projects here:
https://github.com/aspnet/HttpRepl/commit/eb0788c03704abafd1a98436677c081881d7dbec